internal/input/operations.go

```diff
@@ -56,13 +60,17 @@ func GetOperations(start, end uint32, limit int64, env utils.EnvironmentDetails)
 	err = backend.PrepareRange(ctx, ledgerbackend.BoundedRange(start, end))
 	panicIf(err)
 	for seq := start; seq <= end; seq++ {
-		// txReader, err := ingest.NewLedgerTransactionReader(ctx, backend, publicPassword, seq)
 		changeReader, err := ingest.NewLedgerChangeReader(ctx, backend, env.NetworkPassphrase, seq)
 		if err != nil {
 			return []OperationTransformInput{}, err
 		}
 		txReader := changeReader.LedgerTransactionReader
 
+		ledgerCloseMeta, err := backend.GetLedger(ctx, seq)
+		if err != nil {
+			return nil, errors.Wrap(err, "error getting ledger from the backend")
```

Comment:
would it be helpful to return the sequence number with the error so we know where the error occurred?